README.md

```diff
@@ -132,7 +132,7 @@ $ make && bazel run //accelerator:push_docker --host_force_python=PY2
 MQTT connectivity is an optional feature allowing IoT endpoint devices to collaborate with `Tangle-Accelerator`.
 
 ```
-make MQTT && bazel run //accelerator_mqtt
+make && bazel run //accelerator:accelerator_mqtt
```

Comment:
plz help me changing it to `make MQTT && bazel run //accelerator:accelerator_mqtt`

Reply:
Changed.